tests/out/glsl/operators.main.Compute.glsl

```diff
@@ -42,7 +42,7 @@ vec3 bool_cast(vec3 x) {
 }
 
 float constructors() {
-    Foo foo;
+    Foo foo = Foo(vec4(0.0, 0.0, 0.0, 0.0), 0);
```

Comment:
can we make vector initialization use shorter syntax? i.e. `vec4(0.0)` instead of enumerating all components